src/LottoGame.js

```diff
@@ -0,0 +1,91 @@
+import { Console } from "@woowacourse/mission-utils";
+import LottoManager from "./LottoManager.js";
+import { INPUT_MESSAGE, PRINT_MESSAGE } from "./constants/lottoMessage.js";
+import { LOTTO_PRIZE } from "./constants/lottoRule.js";
+
+class LottoGame {
+  constructor() {
+    this.lottoManager = new LottoManager();
+  }
+
+  async start() {
+    const purchaseAmount = await this.getPurchaseAmount();
+    const lottos = this.lottoManager.buyLottos(purchaseAmount);
+    this.printPurchasedLottos(lottos);
+
+    const winningNumbers = await this.getWinningNumbers();
+    const bonusNumber = await this.getBonusNumber(winningNumbers);
+
+    const result = this.lottoManager.checkWinningResult(
+      lottos,
+      winningNumbers,
+      bonusNumber
+    );
+    this.printWinningStatistics(result, purchaseAmount);
+  }
+
+  async getPurchaseAmount() {
+    while (true) {
+      try {
+        const input = await Console.readLineAsync(
+          INPUT_MESSAGE.PURCHASE_AMOUNT
+        );
+        const amount = parseInt(input);
+        this.lottoManager.validatePurchaseAmount(amount);
+        return amount;
+      } catch (error) {
+        Console.print(error.message);
+      }
+    }
+  }
+
+  printPurchasedLottos(lottos) {
+    Console.print(PRINT_MESSAGE.PURCHASE_AMOUNT(lottos.length));
+    lottos.forEach((lotto) => Console.print(`[${lotto.numbers.join(", ")}]`));
+    Console.print("");
+  }
+
+  async getWinningNumbers() {
+    while (true) {
+      try {
+        const input = await Console.readLineAsync(
+          INPUT_MESSAGE.WINNING_NUMBERS
+        );
+        return this.lottoManager.parseWinningNumbers(input);
```

Comment:
```suggestion
        return this.lottoManager.validateWinningNumbers(input);
```
`parse`라는 단어가 유효성 검사를 예측하기는 조금 어렵다고 생각하는데 위에 구매 금액 유효성검사 메서드처럼 `validate` 접두사를 사용해서 일관성과 의도를 전달하는건 어떨까요??

Reply:
parseWinningNumbers 라고 붙인 이유가 처음 split을 통해 parse를 진행해서 이름이 이렇게 나왔던 것 같아요. ! 
이것을 제외하면 validation하는 것이 주된 목적이니 validateWinningNumbers으로 수정하는 것이 좋을 것 같습니다.